### PR TITLE
29 testimonial content type

### PR DESCRIPTION
**Pull request by dharizza**
Added feature for testimonial content type and related permissions. The role drupal_administrator is the one who can add new testimonials.

### CODE REVIEW

**Comment by kporras07** [comment]
Please revert d7ba8e4 (it's giving us some issues because webform module was deleted and it shouldn't be updated)

**Comment by kporras07** [comment]
Now, we have merge conflicts. Please fix it.

**Comment by kporras07** [comment]
Please remove  "Promoted to front page" option in content type options.

**Comment by kporras07** [comment]
Besides that; the functionality looks well :+1:

**Comment by dharizza** [comment]
Ready with your comments.

File: sites/all/modules/custom/dccr/dccr.install [hunk 1]
```diff
@@ -117,9 +117,17 @@ function dccr_update_7004() {
 }
 
 /**
- * Enables dccr_ftr_keynote feature.
+ * Enables dccr_ftr_sessions feature.
  */
 function dccr_update_7005() {
```
Comment (kporras07) [inline sites/all/modules/custom/dccr/dccr.install:121]: Your hook_update_N (new update_N going to master) should have the latest number

**Comment by dharizza** [comment]
Hi, sorry, now it is fixed. Please check.

**Comment by rigoucr** [comment]
You have merge conflicts

**Comment by dharizza** [comment]
Please check again.

**Comment by rigoucr** [comment]
:shipit:  :D

**Pull request merged**